mne/io/edf/edf.py

```diff
@@ -1589,7 +1597,7 @@ def _read_annotations_edf(annotations):
     """
     pat = '([+-]\\d+\\.?\\d*)(\x15(\\d+\\.?\\d*))?(\x14.*?)\x14\x00'
     if isinstance(annotations, str):
-        with open(annotations, encoding='latin-1') as annot_file:
+        with open(annotations, encoding=encoding) as annot_file:
```

Comment:
this is an API change? bug fix?

Reply:
It's the same change applied to loading a file (when passing the filename as a string).

Reply:
I don't really understand what is happening here. The file `mne_data/MNE-testing-data/EDF/test_generator_2.edf` can be loaded with `mne.io.read_raw_edf()` (with the default UTF8 encoding), but `mne.read_annotations()` fails with UTF8 and requires Latin-1...

Reply:
I think I fixed it. For some unknown reason, `read_annotations()` read the EDF file as text instead of binary. This might have worked by accident, but know I'm reading it as binary and I think this is the correct way to do it.